torch/csrc/utils/python_dispatch.cpp

```diff
@@ -171,7 +171,6 @@ class PythonKernelHolder : public c10::OperatorKernel {
 
     auto arguments = torch::jit::pop(*stack, op.schema().arguments().size());
     py::gil_scoped_acquire g;
-    EnableHermeticPyObject g2;
```

Comment:
This change is not acceptable and we need a different way to handle this..

Reply:
I think the two directions we can go here are:
- Thread some flag from torch.library to the kernel to control if this is set or not.
- Make a twisted custom op so that you don't hit this anymore (pass in a fake NestedTensor as input? :p )

Reply:
The second is how I suggested joel workaround it lol

Reply:
okay I added a dummy arg, although it makes me sad